### PR TITLE
fix: reintroduces usd amounts

### LINKED ISSUE #2735: Fix poor wrapping of content account content
Fix poor wrapping of content in account box

### LINKED ISSUE #2756: Restore fiat conversion

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/3376284672).

Reinstates fiat amounts in the list. Also improves handling of narrow viewports. It's not perfect at extremely narrow widths, but not sure we need to optimise for this.

Closes #2756, #2735

https://user-images.githubusercontent.com/1618764/198989372-5ec36b82-1810-4ba5-86b8-d92875a98373.mp4

### CODE REVIEW

File: src/shared/models/money.model.ts [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
 
 export interface Money {
   readonly amount: BigNumber;
   readonly symbol: Currencies;
+  readonly decimals: number;
 }
 
-export function createMoney(amount: BigNumber | number, symbol: Currencies): Money {
-  return { amount: new BigNumber(amount), symbol };
+//
```
Comment (edu-stx) [inline src/shared/models/money.model.ts]: Intended?

**Review by fbwoolf — APPROVED** [review]
Looks good. 👍

**Comment by fbwoolf** [comment]
Any thoughts on how to handle the overlapping checkmark on the active account?

**Comment by markmhendrickson** [comment]
My initial sense is that narrowing the width to around this point should cause the subtext (i.e. address, STX and fiat value line) to shrink with an ellipses (...)

image

And that the shrinkage should occur so there's always about this much padding between that content and the checkmark:

image

Same thing should apply to the account label itself in case it's quite long

**Comment by kyranjamie** [comment]
By popular demand, I've adjusted the layout to better handle extremely small viewports, narrowing margins, and moving the arrow next to the name to prevent overlaps.  

image


I'm reluctant to optimise this further, as I highlight doubt there are users using the wallet in a window narrower than the popup window.

File: src/shared/models/money.model.ts (hunk 1 above)
Comment (kyranjamie) [reply src/shared/models/money.model.ts]: Force of habit, I used to do this stylistically 👀

**Review by edu-stx — APPROVED** [review]

**Pull request merged**